indico/modules/events/timetable/views/weeks.py

```diff
@@ -105,5 +106,8 @@ def inject_week_timetable(event, days, tz_name):
             tmp.append((day, day_entries))
         week_table.append(tmp)
 
-    return render_template('events/timetable/display/_weeks.html', event=event, week_table=week_table,
+    entry_colors = {}
+    if layout_settings.get(event, 'timetable_theme') == 'indico-weeks-view':
```

Comment:
I would rather expose all the theme settings to the template (and then get whatever item you need in the jinja template).
Also, the theme settings dict should be available when rendering a "normal" theme. Otherwise it's only usable for the weeks theme.